static/js/financial_aid/functions.js

```diff
@@ -1,9 +1,27 @@
 /* eslint-disable no-undef, max-len, no-var, prefer-template, no-unused-vars */
-var financialAidReview = function() {
+global.jQuery = require('jquery');
+
+require("bootstrap");
+
+import { codeToCountryName } from "../util/currency";
+
+window.codeToCountryName = codeToCountryName;
+
+/* Wrapper for all of the functions used by the financial aid review page */
+window.financialAidReview = (function(window, document, $) {
```

Comment:
Ok I see - there's no React here, but I assume you mean outside of the webpack minified dealio that ends up being executed.